### ISSUE TITLE
[1.12.1] Magnet mode not always working

### ISSUE THREAD

**Issue by Loothor** [TheCBProject/NotEnoughItems#125]
Hi there,

Just wanted to bring to attention - Magnet mode doesn't always work in this version. Sometimes it pulls items to you, sometimes it pulls items and they _circle_ you (this is probably more annoying that it not working...lol), and most of the time it doesn't pull items at all.

Thanks for reading, and thanks for providing this mod for years - it's always been a part of my servers.

**Comment by DatrixTHLK** [comment]
Hi CBP,

I can confirm this doesn't work at all for 1.12.2 No matter what I try. I went in to the NEI configuration and hit global and clicked on a bunch of options then clicked on Global again. I saw the options on the inventory screen, but it won't magnetize the items to me even though I clicked on the option.

Minecraft: 1.12.2
Forge: 14.23.0.2531
Code Chicken Lib: 3.1.3.313
NEI: 2.4.0.231

Thanks again for at least looking at this.
